src/main.js

```diff
@@ -33,7 +34,7 @@ function errorHandler(error) {
       networkError.message.startsWith('NetworkError')) ||
       failoverOn.includes(networkError.statusCode))
```

Comment:
Do we want apollo queries to only reload on 302s, when we've removed that restriction everywhere else?

Reply:
Not happy to write this: I'm to scared to change this right now.